sklearn/kernel_approximation.py

```diff
@@ -626,6 +630,9 @@ class AdditiveChi2Sampler(TransformerMixin, BaseEstimator):
     This estimator approximates a slightly different version of the additive
     chi squared kernel then ``metric.additive_chi2`` computes.
 
+    Using `fit_transform` is preferable over using `transform` to benefit
+    from parameters validation.
```

Comment:
To be consistent with the changes in #25191 

```suggestion
    This estimator is stateless and does not need to be fitted. However, we
    recommend to call :meth:`fit_transform` instead of :meth:`transform`, as
    parameter validation is only performed in :meth:`fit`.
```